packages/common-settings-bridge/src/__mocks__/matrix-appservice-bridge.ts

```
@@ -0,0 +1,76 @@
+        isSelfAdmin: jest.fn().mockResolvedValue(true),
+        upsertUser: jest.fn().mockResolvedValue(undefined)
+      }
+    },
+    uploadContentFromUrl: jest
+      .fn()
+      .mockResolvedValue('mxc://example.com/avatar123')
+  }
+
+  constructor() {
+    // Return the shared mock intent instance
+    return mockIntent as any
```

Biome (2.3.13)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor may confuse users of the class.

(lint/correctness/noConstructorReturn)